plugins/node/opentelemetry-instrumentation-aws-sdk/test/load-instrumentation.ts

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I tried many attempts to have the meter-enabled instrumentation running for bedrock tests, including stuff like `beforeEach(() => registerInstrumentationForTesting(); instrumentation.enable())` but nothing worked, so I ended up refactoring to make sure there's only a single instrumentation instance among all these tests (since mocha doesn't isolate tests)

Reply:
Yah, mocha running all tests in the same single process is a pain, IMHO. (Test frameworks often go this way though because speed.)

Could you add a block comment at the top of the file (after the license block) that explains basically what you've said above.